debian/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM debian
+
+RUN apt-get update && apt-get install -y curl python2.7 apt-transport-https lsb-release
+RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
+    echo "deb https://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
```

Comment:
1.  why do you need `tee`? can't you just `echo .. > ..`?
2. why do you need `-a`, is there already anything at `/etc/apt/sources.list.d/google-cloud-sdk.list`?